src/stories/Library/search-result-page/ResultPager.tsx

```diff
@@ -1,15 +1,16 @@
 import { Button } from "../Buttons/button/Button";
 
 interface ResultPagerProps {
-  currentResults: string;
-  totalResults: string;
+  currentResults: number;
+  totalResults: number;
 }
 
 const ResultPager = ({ currentResults, totalResults }: ResultPagerProps) => {
+  const total = totalResults.toLocaleString("da-Dk");
```

Comment:
This line is probably here because of commas and dots in connection to thousands. Again, great thinking.